lib/renaissance_web/controllers/auction_controller.ex

```diff
@@ -0,0 +1,42 @@
+defmodule RenaissanceWeb.AuctionController do
+  use RenaissanceWeb, :controller
+  alias Renaissance.{Auction, Auctions}
+  alias RenaissanceWeb.Helpers.{Auth}
+
+  def new(conn, _params) do
+    changeset = Auction.changeset(%Auction{})
+
+    if Auth.signed_in?(conn) do
+      render(conn, "new_auction.html", changeset: changeset)
+    else
+      conn
+      |> redirect(to: Routes.login_path(conn, :login))
+    end
+  end
+
+  def create(conn, params) do
+    changeset =
```

Comment:
This isn't a changeset.

Reply:
Resolved with 159a775e493da7136d042a4f76b690b4d8cb2aea